Automattic/jetpack
# Recommendations: update link in summary upsell

Fixes https://github.com/Automattic/jetpack/issues/32461

## Proposed changes:

The call-to-action in the summary page of the Recommendations section links to a broken checkout page, since no product is specified. This PR updates the link with the pricing page URL.

### Other information:


## Jetpack product discussion


p1704733531298729-slack-C01264051NE

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- Build the plugin
- Connect the site without purchasing any subscriptions
- Visit the Recommendations section
<img width="500" alt="Screenshot 2024-01-08 at 11 30 59 AM" src="https://github.com/Automattic/jetpack/assets/1620183/ba614bfa-9a25-4a27-8374-2a97837f4541">

- Complete the wizard until you arrive at the summary step
<img width="500" alt="Screenshot 2024-01-08 at 11 30 54 AM" src="https://github.com/Automattic/jetpack/assets/1620183/358afd8a-9caf-4d9f-911a-1ba675e056fa">

- Verify that the _Learn more_ call-to-action links to https://cloud.jetpack.com/pricing/
